students/jakielczyk_miroslaw/lesson_09_reading_and_writing_files/folder_with_text_files/text_file_nr1.txt

```diff
@@ -0,0 +1,5 @@
+kody pocztowe plik nr1
```

Comment:
I think this file belong to previous lesson.
Please remove it from this PR

Reply:
folder "folder_with_text_files" has been created on purpose, inside this folder there are two txt files, for these files regex_expression is further used

Reply:
ok, got it